documentation/build/html/_sources/antisphinx.txt

```diff
@@ -45,7 +45,6 @@ File Layout
 ***********
```

Comment:
Sphinx copies the .rst files it processes to /_sources/ as a txt file to make them available as source code on the documentation (through a link in the top right corner)